src/index.html

```diff
@@ -1,16 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form"
+      data-qa="big"
+    >
```

Comment:
please fix attributes formatting: each attribute shold be in the new line

![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/91946036/07716e5c-f2ef-402b-a793-9b6f2ff7b706)

[Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

Reply:
mb I don't understand the formatting principle very well... but ok, I'll fix it